app/views/content/non-uk-teachers/return-to-england-after-teaching-overseas/_check-your-qualifications.md

```diff
@@ -1,8 +1,8 @@
-You will need [qualified teacher status (QTS)](https://www.gov.uk/guidance/qualified-teacher-status-qts) to teach in a maintained school in England.
+[Qualified teacher status (QTS)](https://www.gov.uk/guidance/qualified-teacher-status-qts) is a legal requirement to teach in many English schools, and considered desirable for teachers in the majority of schools in England.
```

Comment:
```suggestion
[Qualified teacher status (QTS)](https://www.gov.uk/guidance/qualified-teacher-status-qts) is a legal requirement to teach in many English schools.
```
Not sure we need the second half of the sentence if further down we're saying 'however, even where QTS is not a legal requirements, many schools use QTS to assess the quality of candidates for teaching jobs.'